### DISCUSSION TITLE
External cluster script for Rook in Kubernetes

### DISCUSSION THREAD

**Discussion by andwards** [rook/rook#13217] [Q&A]
I am having an issue while trying to extract information from an [external cluster](https://rook.io/docs/rook/v1.12/CRDs/Cluster/external-cluster/#prerequisites) to connect it to Kubernetes using Rook. Specifically, I am running the Python script [create-external-cluster-resources.py](https://github.com/rook/rook/blob/release-1.12/deploy/examples/create-external-cluster-resources.py) provided by Rook and passing in the required arguments and Ceph configured with these components:

`python3 create-external-cluster-resources.py --rbd-data-pool-name ceph-rbd --cephfs-filesystem-name cephfs --namespace rook-ceph-external --format bash`

However, I am encountering the following error:

```
Traceback (most recent call last):
  File "create-external-cluster-resources.py", line 1, in <module>
    {"payload":{"allShortcutsEnabled":false,"fileTree":{"deploy/ … ,"title":"rook/deploy/examples/create-external-cluster-resources.py at release-1.12 · rook/rook"}
NameError: name 'false' is not defined
```

Unfortunately, the payload is too extensive to fully include here.

I appreciate any suggestions/help

**Comment by subhamkrai** [comment]
@andwards what version of the Python script you are using 1.12?

can you share the complete error?
